test/evergreen.yml

```diff
@@ -2113,7 +2113,7 @@ tasks:
           # Make sure we dump core on failure
           format_test_setting: ulimit -c unlimited
           #run for 2 hours ( 2 * 60 = 120 minutes), use default config
-          format_test_script_args: -b "SEGFAULT_SIGNALS=all catchsegv ./t" -t 120
+          format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 120
```

Comment:
I'm confused why `-e` is useful, can't we just set `"SEGFAULT_SIGNALS=all"` in the environment before running the script?

Reply:
I assume yes - we can set the env variable if running format.sh standalone. The `-e` can make the code neat in evergreen.yml (not all tasks calling `format test script` evergreen function need to set the env variable).